addons/binding/org.openhab.binding.fritzaha/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fritzaha"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 org.eclipse.smarthome.thing-description.xsd">
+
+	<bridge-type id="fritzaha-bridge">
+		<label>Fritz!Box AHA</label>
+		<description>This bridge represents a Fritz!-Box as gateway to connected AHA devices.
+		</description>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>Fritz!Box LAN IP</label>
+				<description>The IP address or hostname of the Fritz!Box
+				</description>
+				<default>fritz.box</default>
+			</parameter>
+			<parameter name="port" type="integer" required="false" min="0"
+				max="65335">
+				<context>network-address</context>
+				<label>Fritz!Box port</label>
+				<description>Port of the Fritz!Box</description>
+				<default>0</default>
+			</parameter>
+			<parameter name="protocol" type="text" required="false">
+				<context>network-address</context>
+				<label>Fritz!Box protocol</label>
+				<description>Protocol to connect to the Fritz!Box (http or https)</description>
+				<default>http</default>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<context>authentication</context>
+				<label>Fritz!Box password</label>
+				<description>Password to access the Fritz!Box device.
+				</description>
+			</parameter>
+			<parameter name="user" type="text" required="false">
+				<context>authentication</context>
+				<label>Fritz!Box user</label>
+				<description>User name which has HomeAuto permissions on the given Fritz!Box.
+				</description>
+				<default></default>
```

Comment:
this should be removed and the code should be able to handle null when a parameter is not required